README.md

```diff
@@ -1,86 +1,104 @@
 
-| Feature          | iOS     | Android | UWP     | WinUI   |
-| ---------------- | ------- | ------- | ------- | ------- |
-| GetCapability    | &check; | &check; | &check; | &check; |
-| ValidateIdentity | &check; | &check; | &cross; | &cross; |
-| Encrypt          | &check; | &check; | &cross; | &cross; |
-| Decrypt          | &check; | &check; | &cross; | &cross; |
+   - Please note that Encrypt/Decrypt methods are only available when using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)` in the `BiometricPrompt.PromptInfo.Builder` instantiation that is required for the service.
 
-## Getting Started
+   - Please also note that the title and subtitle are used for `Fingerprint` and `Face` biometry.
 
-Install the latest stable version of `BiometryService` in your platform heads and `BiometryService.Abstractions` in your presentation layer if you are using MVVM pattern, and if not just install both in your platform heads.
+   - Here is an example of instantiation of the service for Android.
 
-A small sample Uno application is available as a playground with some basic command to test the service methods.
-They also provide some basic initialization but no dependency injection and more complex code.
+   ``` cs
+   private readonly IBiometryService _biometryService;
+   //(...)
+   var promptBuilder = () => new BiometricPrompt.PromptInfo.Builder()
+	   .SetTitle("Title")
+	   .SetSubtitle("Subtitle")
+	   .SetNegativeButtonText("Cancel")
+	   .SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)
+	   .Build();
 
-### Instantiation
+   var biometryService = new BiometryService(
+   	   fragmentActivity: MainActivity.Instance,
+   	   promptInfoBuilder: promptBuilder,
+   	   loggerFactory: null
+   );
+   ```
+   ### iOS
 
-#### Android
+   - Please note that you must set `NSFaceIDUsageDescription` (key/value) in the `Info.plist` file otherwise the service will throw an exception.
 
-Face authentication is only available when using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricWeak)` in the `BiometricPrompt.PromptInfo.Builder` instantiation that is required for the service. Please note that if you are using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)` in the `BiometricPrompt.PromptInfo.Builder` Face authentication is only available on a Google Pixel 4 as of now.
+   ``` xml
+    <!-- info.plist -->
```

Comment:
Here I think you can keep only the `key` and `string` tags. People will understand the Info.plist context even without the full xml hierarchy.